packages/tokens/src/alias/teamsDarkColor.ts

```diff
@@ -38,7 +38,9 @@ export const generateColorTokens = (brand: BrandVariants): ColorTokens => ({
   colorCompoundBrandForeground1Hover: brand[110], // #3aa0f3 Global.Color.Brand.110
   colorCompoundBrandForeground1Pressed: brand[90], // #1890f1 Global.Color.Brand.90
   colorBrandForeground1: brand[100], // #2899f5 Global.Color.Brand.100
-  colorBrandForeground2: brand[120], // #6cb8f6 Global.Color.Brand.120
+  colorBrandForeground2: brand[110], // #3aa0f3 Global.Color.Brand.110
```

Comment:
@redongreen - `colorBrandForeground2` in Teams Dark theme was previously changed from Brand.110 to Brand.120 to fix #23898. Are we sure we want to assign this back to 110?

Reply:
After discussion with @daisygeng reverting back to `brand[120]` in Teams Dark.